wlauto/workloads/geekbench/__init__.py

```diff
@@ -87,6 +87,14 @@ class Geekbench(AndroidUiAutoBenchmark):
         Parameter('times', kind=int, default=1,
                   description=('Specfies the number of times the benchmark will be run in a "tight '
                                'loop", i.e. without performaing setup/teardown inbetween.')),
+        Parameter('iteration_timeout', kind=int, default=900,
```

Comment:
Rename to just `timeout` -- for consistency with other workloads; also the term "iteration" should not be applied to the "times" parameter as it means something else in the context of a WA run.